### PR TITLE
ARUHA-655: add zappr for compliance

### PR DESCRIPTION
**Pull request by rcillo**
https://techjira.zalando.net/browse/ARUHA-655

### CODE REVIEW

**Comment by lmontrieux** [comment]
👍

File: .zappr.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+approvals:
+  minimum: 1
+  ignore: pr_opener
+  pattern: "^(👍|:\\+1:|:thumbsup:|:shipit:)$"
+  veto:
+    pattern: "^(👎|:-1:|:thumbsdown:)$"
+  from:
+    users:
```
Comment (v-stepanov) [inline .zappr.yaml:8]: what about @dehora ?

Comment (rcillo) [reply .zappr.yaml:8]: Good. I forgot that it's fine to have someone from not formally inside the team.

**Comment by rcillo** [comment]
@v-stepanov @lmontrieux added @dehora as approver.

**Comment by lmontrieux** [comment]
👍

**Comment by v-stepanov** [comment]
👍

**Pull request merged**